app/src/main/java/org/android/go/sopt/presentation/login/LoginViewModel.kt

```diff
@@ -1,6 +1,40 @@
 package org.android.go.sopt.presentation.login
 
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import kotlinx.coroutines.launch
+import org.android.go.sopt.data.model.main.User
+import org.android.go.sopt.data.model.request.RequestSignInDto
+import org.android.go.sopt.domain.AuthRepository
+import javax.inject.Inject
 
-class LoginViewModel : ViewModel(){
+@dagger.hilt.android.lifecycle.HiltViewModel
+class LoginViewModel @Inject constructor(private val apiAuthRepository: AuthRepository): ViewModel(){
```

Comment:
```suggestion
import dagger.hilt.android.lifecycle.HiltViewModel

@HiltViewModel
class LoginViewModel @Inject constructor(
    private val apiAuthRepository: AuthRepository
) : ViewModel() {
```
import와 code reformat을 통해 코드 가독성을 개선해주면 좋을 것 같습니다!